.env

```diff
@@ -0,0 +1,25 @@
+# Django Secret Key
+APP_SECRET_KEY = 'django-insecure-mxd#*-dutjdhw5_4-39*0nfz00#5cuntlr*uuqhktvqjurdt9_'
+
+# Making Python aware of where to start
+PYTHONPATH=/path/to/the/repository/
```

Comment:
This path leads to nowhere. Are you sure this is the path to the project directory on Docker?